js/packages/core/test/it/ufg/open-eyes.spec.ts

```diff
@@ -22,4 +22,45 @@ describe('open-eyes', () => {
       error => error.message === 'get account info failed',
     )
   })
+
+  it('Should pass test information correctly', async () => {
```

Comment:
I would call it `populate`, not `pass`
```suggestion
  it('should populate test information', async () => {
```